src/genericBlogPost.tsx

```diff
@@ -0,0 +1,143 @@
+import * as React from 'react'
+import {
+  getConfig,
+  Actions,
+  State,
+  getPages,
+  getOpenGraph,
+  getBlogPosts,
+  getAuthors,
+  getBlogPostMarkdownBlocks,
+  hasTag,
+} from './store'
+import { Page } from './store/websiteReducer'
+import { Subscribe } from './layout.v2'
+import { Head, Html, Body, Navbar, OpenGraph, Footer } from './layout.v3'
+import { format } from 'date-fns'
+import { join } from 'path'
+import { read } from './files'
+import { JsonLd } from 'react-schemaorg'
+import { BlogPosting } from 'schema-dts'
+import { renderToJsx, toMdast } from './markdown'
+import { Author } from './article.v2'
+import { transform } from './markdown/utils'
+import { selectAll } from 'unist-util-select'
+import * as Mdast from 'mdast'
+import { mdast2Jsx } from './markdown/jsx'
+import { defaultAssetsPipeline } from './optimise'
+import { Store } from 'redux'
+
+export async function Mount({ store }: { store: Store<State, Actions> }) {
+  const state = store.getState()
+  const pages = getPages(state).filter(hasTag(state, 'blog-post'))
```

Comment:
I think this could have been:

```ts
getPages(state).filter(it => !hasTag(state, 'bite-sized')(it))
```

In other words, all the blog posts that are not part of _bite-sized_.